src/insertion_sort.py

```diff
@@ -0,0 +1,52 @@
+
+"""Bubble sort a list."""
+import time
+from random import randint
+
+
+def insert_sort(list):
+    """Bubble sort a list."""
+    for i in range(len(list) - 1):
+        sort = True
+        while sort:
+            if list[i] > list[i + 1]:
+                list[i], list[i + 1] = list[i + 1], list[i]
+                if i > 0:
+                    i -= 1
+                else:
+                    sort = False
+            else:
+                sort = False
+    return list
+
+
+if __name__ == '__main__':
```

Comment:
How is the code in this `if __name__ == "__main__"` block showing me the timing differences between the best, worst, and average case scenarios?